bootstrap-pnut-exe.sh

```diff
@@ -4,8 +4,10 @@ set -e -u
 
 TEMP_DIR="bootstrap-results"
 
-PNUT_EXE_OPTIONS= # Set by the backend option
-PNUT_SH_OPTIONS="-DRT_NO_INIT_GLOBALS -Dsh"
+: ${PNUT_OPTIONS:=} # Default to empty options
```

Comment:
The `bootstrap-pnut*.sh` and `run-tests.sh` scripts now passes the `PNUT_OPTIONS` environment to pnut. This allows us to enable certain options (by defining macros) from outside the script.